### DISCUSSION TITLE
Requesting Example .txt File in Root Folder

### DISCUSSION THREAD

**Discussion by kaylaharrison** [CMSgov/price-transparency-guide#725] [Q&A]
One of the accessibility requirements for 01/01/24 is to include a .txt file in the root folder of the public website. Can an example of this be shared to ensure we are meeting CMS requirements?

**Comment by shaselton-usds** [comment]
Hi @kaylaharrison 

The .txt file is a [Hospital Price Transparency](https://github.com/CMSgov/hospital-price-transparency) requirement. It looks like you opened a discussion over there so I'll go ahead and close this discussion out..

**Reply by kaylaharrison** [reply]
Hi,

I understand that it is part of the price transparency requirement. But what does “placing a .txt file in the root folder of the public website” mean? We were hoping we could get an example of what the .txt file should look like.

Thank you,
